plugins/communications-app/BodyForm/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Form } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import TextEditor from '@communications-app/src/components/bulk-email-tool/text-editor/TextEditor';
+import { useSelector, useDispatch } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context';
+import { actionCreators as formActions } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context/reducer';
+
+import messages from './messages';
+
+const BodyForm = () => {
+  const intl = useIntl();
+  const formData = useSelector((state) => state.form);
+  const dispatch = useDispatch();
+  const { body, isFormSubmitted = false } = formData;
+
+  const handleChangeTextEditor = (value) => {
+    dispatch(formActions.updateForm({ body: value }));
+  };
+
+  const bodyValidation = body.length > 0;
```

Comment:
What about this:

```suggestion
  const isBodyValid = body.length > 0;
```

Or

```suggestion
  const isFormValid = body.length > 0;
```

I consider that more declarative for booleans.